pywbem/_cim_operations.py

```diff
@@ -1599,6 +1602,39 @@ def _activate_logger(cls, logger_name, simple_name, detail_level, handler,
                 # TODO execute stage of wbem connection if not already executed.
                 # How do we know it is already executed???
 
+    def _verify_open(self):
+        """
+        Verify that this connection is open and raise ConnectionError otherwise.
+
+        Raises:
+          :exc:`~pywbem.ConnectionError`: Connection must be open but is closed
+        """
+        if self.session is None:
+            raise ConnectionError(
+                "Connection must be open but is actually closed",
```

Comment:
Maybe simpler message. We do not state that connection must be open.  Is there any case where self.session is marked none by something other than our close.  If not message could be "connection closed by pywbem"

Reply:
We only set self.session = None in case of closing the connection.
I agree that we could simplify the message, I have changed it to "WBEMConnection is closed", to distinguish this from the general term "connection".